framework/auth/core.py

```diff
@@ -210,7 +215,7 @@ class User(GuidStoredObject, AddonModelMixin):
         'github': u'http://github.com/{}',
         'scholar': u'http://scholar.google.com/citation?user={}',
         'twitter': u'http://twitter.com/{}',
-        'personal': u'{}',
+        'profileWebsites': u'[{}]',
```

Comment:
Thanks @fabianvf .

@billyhunt Have you checked how this affects the display of search results?